app/lib/app/SecvisogramPage/View/WizzardEditorTab/WizardPanel/editors/GenericEditor/ArrayEditor.js

```diff
@@ -38,27 +28,91 @@ export default function ArrayEditor({ property, instancePath }) {
   return (
     <>
       <div className="border-l border-r border-solid bg-gray-50 border-gray-400 wizard-menu-shadow mr-2">
-        <ul>
-          {sanitizedValue.map((_, i) => {
-            const indexErrors = errors.filter(
-              (e) => e.instancePath.startsWith('/' + [...instancePath, i].join('/'))
-            )
-            return (
-              <li key={instancePath.concat([String(i)]).join('.')}
-                  className={`${selectedIndex === i
-                    ? 'border-l-4 border-blue-400 border-b border-gray-300'
+        <Menu
+          value={sanitizedValue}
+          instancePath={instancePath}
+          property={property}
+        />
+      </div>
+      {selectedSubPath.length ? (
+        <GenericEditor
+          key={selectedPath.join('.')}
+          parentProperty={property}
+          property={childProperty}
+          instancePath={selectedPath}
+        />
```

Comment:
Selecting a nested path in the tree (`full_product_name.product_identification_helpers`) seems not to open the editor for it anymore.